core/src/main/java/hudson/FilePath.java

```diff
@@ -77,16 +77,11 @@
 import java.net.URI;
 import java.net.URL;
 import java.net.URLConnection;
-import java.nio.file.Files;
-import java.nio.file.InvalidPathException;
-import java.nio.file.StandardCopyOption;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Comparator;
-import java.util.Enumeration;
-import java.util.List;
-import java.util.Map;
-import java.util.StringTokenizer;
+import java.nio.file.*;
```

Comment:
nit: I prefer to avoid `.*` imports to make it clear exactly what classes are being used.